src/main/java/org/gridsuite/odre/server/utils/GeographicDataParser.java

```diff
@@ -119,9 +114,13 @@ public static Map<String, LineGeoData> parseLines(BufferedReader aerialLinesBr,
         stopWatch.start();
 
         Map<String, Graph<Coordinate, Object>> graphByLine = new HashMap<>();
+        Map<String, String> idsColumnsName = new HashMap<>(
```

Comment:
those maps are statics (never change). Then put it otherwhere in the code, then use it directly in parseLine function, it doesn't need to be passed as function arguments